<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет оценок эффективности ЭСМ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Точечная оценка</string>
     </attribute>
     <widget class="QPushButton" name="buttonOpenFileBasePeriodValues">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textboxBasePeriodValuesFilename">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Значения базового периода:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Значения отчетного периода:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textboxBaseReportValuesFilename">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonOpenFileReportPeriodValues">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonReset1">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Единица измерения целевой переменной:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="listUnits1">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>160</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>кВт</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>МВт</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Гкал</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Целевая переменная:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="listVariables">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonGetPointEstimate">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>200</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти оценку</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Эффективность в натуральном выражении:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelEfficiencyScore">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>200</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Интервальная оценка</string>
     </attribute>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Значения потребления с учетом ЭСМ:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Единица измерения целевой переменной:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonOpenFileIntervalForecast">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textboxActualValuestFilename">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonOpenFileActualValues">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>150</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonReset2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textboxIntervalForecastFilename">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Значения потребления без учета ЭСМ:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="listUnits2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>кВт</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>МВт</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Гкал</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Верхняя оценка (название переменной):</string>
      </property>
     </widget>
     <widget class="QComboBox" name="listVariablesUpper">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>60</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Нижняя оценка (название переменной):</string>
      </property>
     </widget>
     <widget class="QComboBox" name="listVariablesLower">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>90</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="listActualsVariable">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>190</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelIntervalEfficiencyUpperScore">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonGetIntervalEstimate">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>290</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти оценку</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>501</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Эффективность в натуральном выражении (верхняя граница):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Целевая переменная:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>501</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Эффективность в натуральном выражении (нижняя граница):</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelIntervalEfficiencyLowerScore">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>320</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
